src/main/java/chess/domain/ChessGame.java

```diff
@@ -0,0 +1,52 @@
+package chess.domain;
+
+import chess.domain.piece.Piece;
+
+import java.util.Map;
+
+public class ChessGame {
+    private final Board board;
+    private Team turn;
+
+    public ChessGame() {
+        this.board = new Board();
+    }
+
+    public Map<Position, Piece> start() {
+        if (turn != Team.WHITE && turn != Team.BLACK) {
```

Comment:
```suggestion
        if (turn.notStarted()) {
```
와 같이 메서드를 추가해 볼 수 있겠네요!

Reply:
'ChessGame의 차례가 시작하지 않았다' 보다 'ChessGame이 시작하지 않았다' 라는 의미가 더 적절하다는 생각이 들어 turn.notStarted() 대신 notStarted() 로 추상화 하였습니다!